lib/ecl/ecl_grid.cpp

```diff
@@ -6854,6 +6854,13 @@ static void ecl_grid_fwrite_EGRID__( ecl_grid_type * grid , fortio_type * fortio
 
 void ecl_grid_fwrite_EGRID2( ecl_grid_type * grid , const char * filename, ert_ecl_unit_enum output_unit) {
   bool fmt_file        = false;
+  {
+    bool is_fmt;
+    int rpt_nr = -1;
```

Comment:
This can be removed and then you can just pass a `NULL`-pointer instead.

Reply:
> This can be removed and then you can just pass a `NULL`-pointer instead.

Okay, thanks for the hint.  I'll update accordingly.